Makefile

```diff
@@ -30,6 +34,10 @@ functionaltest: ## 🧪 Run the functional tests
 	@echo -e "\e[34m$@\e[0m" || true
 	@ poetry run pytest -m "functional"
 
+uitest: ## 🧪 Run the ui tests in headless mode
+	@echo -e "\e[34m$@\e[0m" || true
+	@cd code/tests/integration/ui && npm install && npx cypress run --env ADMIN_WEBSITE_NAME=$(ADMIN_WEBSITE_NAME),FRONTEND_WEBSITE_NAME=$(FRONTEND_WEBSITE_NAME)
```

Comment:
Cypress only reads OS environment variables directly if they are prefixed with CYPRESS_, which I don't think we want to do with our outputted env vars since it will tie them to Cypress via their names.

The child shell is able to read the variables, which is why  $(ADMIN_WEBSITE_NAME) is working - but we just need to explicitly pass them on to cypress via the CLI because they aren't prefixed with CYPRESS_.

Reference: https://docs.cypress.io/guides/guides/environment-variables#Setting